Framework/Algorithms/src/EstimateMuonAsymmetryFromCounts.cpp

```diff
@@ -132,6 +156,23 @@ void EstimateMuonAsymmetryFromCounts::exec() {
   // Do the specified spectra only
   int specLength = static_cast<int>(spectra.size());
   std::vector<double> norm(specLength, 0.0);
+
+  double normConst = getProperty("NormalizationIn");
+  for (int j = 0; j < specLength; j++) {
+    if (normConst == 0) {
```

Comment:
This can be changed so the if statements are not inside the loop, it's better not to do the check every time.

Reply:
I'd suggest either store the bool beforehand or use two loops, one for the `if` and one for the `else`.